docs/data-model.md

```diff
@@ -0,0 +1,78 @@
+# Data model description and trait status explanation
+
+- `traits` (id, trait_name, current_mapping[foreign_key], status[computed_key], number_of_source_records, timestamp_added, timestamp_updated)
+
+The master status is meant to reflect the state of the mapping and provide insight as to what should be the next action. It can contain all the possible trait statuses, as well as `unmapped` and in `in_review` statuses. It is never changed explicitly and is always computed based on:
```

Comment:
Starting a separate paragraph in this context confuses Markdown, and causes it to break the list structure. I think it's better to restructure this so that column definitions are first-level items, and all comments and explanations are second (and so on) levels, something like this:

```markdown
- `traits` (id, trait_name, ...)
  + The master status is meant to...
- `users` (...)
- `mappings` (...)
  + The `is_reviewed` field status is...
```